common/utils.cpp

```diff
@@ -46,7 +46,7 @@ uint32_t guessFourcc(const char* fileName)
     const char* extension = strrchr(fileName, '.');
     if (extension) {
         extension++;
-        for (int i = 0; i < N_ELEMENTS(possibleFourcc); i++) {
+        for (uint32_t i = 0; i < N_ELEMENTS(possibleFourcc); i++) {
```

Comment:
ok,agree this. maybe we need find all N_ELEMENTS and change the index to size_t except codec parsers.